velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -338,5 +338,25 @@ TEST_F(ArithmeticTest, hypot) {
   EXPECT_DOUBLE_EQ(5.70087712549569, hypot(3.5, -4.5).value());
 }
 
+class LogNTest : public SparkFunctionBaseTest {
+ protected:
+  std::optional<double> log2(std::optional<double> arg) {
+    return evaluateOnce<double>("log2(c0)", arg);
+  }
+
+  std::optional<double> log10(std::optional<double> arg) {
+    return evaluateOnce<double>("log10(c0)", arg);
+  }
+};
+
+TEST_F(LogNTest, logN) {
```

Comment:
It would be more readable to split this test into 2: log2 and log10, then use local lambdas.

Reply:
Just fixed it.